about.html

```diff
@@ -13,52 +13,100 @@
+
+        <div class="child2-1"> <!--make flexbox column parent-->
+
+          <img src="img/about-plan.png" alt="strategy">
+          
+          <br><h2>Strategy</h2>
+          
+          <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p><br>
+
+          <a href="#">Learn More</a>
+
+        </div>
+
+        <div class="child2-2"> <!--make flexbox column parent-->
+
+          <img src="img/about-working.png" alt="strategy">
+          
+          <br><h2>How We Work</h2>
+          
+          <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p><br>
+
+          <a href="#">Learn More</a>
+
+        </div>
 
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+      </section>
 
-    Learn More
+      <section class="section3"> <!--make flexbox row parent-->
 
+        <div class="child3-1"> <!--make flexbox column parent-->
 
+          <img src="img/about-office.png" alt="strategy">
+          
+          <br><h2>Places We Work</h2>
 
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+          <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p><br>
 
-    Learn More
+          <a href="#">Learn More</a>
 
+        </div>
 
+        <div class="child3-2"> <!--make flexbox column parent-->
 
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
+          <img src="img/about-meeting.png" alt="strategy">
+          
+          <br><h2>Collaboration</h2>
+          
+          <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p><br>
 
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+          <a href="#">Learn More</a>
 
-    Learn More
+        </div>
 
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+      </section>
 
-    Learn More
+      <section class="section4"> <!--make flexbox column parent-->
+      
+        <h2>Let's Work Together</h2>
 
-    Let's Work Together
+        <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+       
+      </section>
 
```

Comment:
Though <br> does create a line break they are most helpful in areas like address, the reason being is because <br> tags are not dry code. 
Dry = Do not repeat yourself
You also lose control of the ability to change the spacing later if there is a style change.